sdk/src/utils/pdf_utils.rs

```diff
@@ -61,7 +62,10 @@ pub(crate) trait C2paPdf: Sized {
     fn write_manifest_as_annotation(&mut self, vec: Vec<u8>) -> Result<(), Error>;
 
     /// Returns a reference to the C2PA manifest bytes.
-    fn read_manifest_bytes(&self) -> Result<Option<Vec<&[u8]>>, Error>;
+    #[allow(clippy::needless_lifetimes)] // required for automock::mockall
+    fn read_manifest_bytes<'a>(&'a self) -> Result<Option<Vec<&'a [u8]>>, Error>;
```

Comment:
the `mockall` macro requires the lifetime to be explicitly defined

Reply:
Appreciate the documentation about the `allow` here.